modules/minimum-required-permissions-ipi-gcp-xpn.adoc

```diff
@@ -7,9 +7,12 @@
 
 When you are installing a cluster to a link:https://cloud.google.com/vpc/docs/shared-vpc[shared VPC], you must configure the service account for both the host project and the service project. If you are not installing to a shared VPC, you can skip this section.
 
-You must apply the minimum roles required for a standard installation as listed above, to the service project. Note that custom roles, and therefore fine-grained permissions, cannot be used in shared VPC installations because GCP does not support adding the required permission `compute.organizations.administerXpn` to custom roles.
+[IMPORTANT]
+====
+You must apply the minimum roles required for a standard installation to the service project.The "Required minimal permissions" table lists these permissions.
+====
 
-In addition, the host project must apply one of the following configurations to the service account:
+Ensure that host project applies one of the following configurations to the service account:
```

Comment:
QUERY TO SME: This statement does not make sense to me. We have a preceding statement that states the minimal permissions are mandatory and this statement says it's optional if one wants to create a firewall. The firewall permissions are required only for a certain situation, while the minimum permissions are alwats required?

Reply:
QUERY TO SME: How does a customer check that the "host project applies one of the following configurations to the service account:"